src/openforms/js/components/admin/form_design/Tab.js

```diff
@@ -0,0 +1,32 @@
+import {useIntl} from 'react-intl';
+import {Tab as ReactTab} from 'react-tabs';
+import FAIcon from '../FAIcon';
```

Comment:
```suggestion
import FAIcon from 'components/admin//FAIcon';
```
& we can group the imports a bit nicer (3rd party / absolute imports / relative imports)